.travis.yml

```diff
@@ -52,8 +52,10 @@ install:
 
 # Run the tests
 script:
+  - mkdir -p "$HOME/.ipython/profile_default"
+  - "echo 'c.HistoryAccessor.enabled = False\n' > $HOME/.ipython/profile_default/ipython_config.py"
```

Comment:
I just copied the line that used to echo `Agg` to `matplotlibrc` and changed the details; I recall having to use quotes there because of how TravisCI parses YAML but I don't know if it's still necessary, or if it ever was